src/patient-registration/patient-registration-helper.tsx

```diff
@@ -19,6 +19,8 @@ interface AttributeValue {
   value: string;
 }
 
+export type Relationships = { uuid: string; name: string; type: string };
+export type tempRelationship = { personA: string; relationshipType: string; personB: string };
```

Comment:
`Relationships` should be singular, since it is a single thing, not an array. I would maybe prefer calling it `SavedRelationship`, to capture exactly what it is that makes a temp relationship temporary.

`tempRelationship` should have an initial capital, because it is a type.

So `SavedRelationship` and `TempRelationship` are my suggestions.

Reply:
Oh wait, now that I'm reading the code below, I see you're doing things like

```jsx
personA: relationship.uuid,
```

This would suggest that `Relationships`/`SavedRelationship` should actually be called something like `RelatedPerson`. That's what that data actually describes. A person. Right?

Reply:
This is also confusing to me. I think it's actually describing a relationship and not a person. I'm talking with respect to the `type` field in the interface definition.

Reply:
I think it's more clearer if the definition looked like

```javascript
export type Relationship = { relatedPersonUuid: string; relatedPersonName: name; relationshipType: string }
```

What do you think @brandones ?

Reply:
Okay. Let's at least capitalize `tempRelationship` to `TempRelationship`.